vocabsieve/forvo.py

```diff
@@ -133,14 +133,15 @@ def get_pronunciations(self) -> Forvo:
                                                  )
 
             self.pronunciations.append(pronunciation_object)
+        self.pronunciations = sorted(self.pronunciations, key=lambda x: x.votes, reverse=True)
```

Comment:
I think it's more useful to always have a sorted list by votes so I've moved the logic here out of `fetch_audio_best`